Manifest.toml

```diff
@@ -149,7 +149,7 @@ version = "0.5.4"
 deps = ["LinearAlgebra", "Markdown"]
 git-tree-sha1 = "8d61b42fe77636ace85083e2420abd89babb9b3e"
 uuid = "3362f125-f0bb-47a3-aa74-596ffd7ef2fb"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
Manifest.toml files need to be updated through `] resolve` because manual changing of version does not update `git-tree-sha1`.

Reply:
I'll fix this.